docs/api/apiv3/components/schemas/storage_model.yml

```diff
@@ -0,0 +1,59 @@
+# Schema: StorageModel
+---
+type: object
+required:
+  - id
+  - _type
+  - href
+  - storageType
+properties:
+  id:
+    type: integer
+    description: Storage id
+  _type:
+    type: string
+    enum:
+      - Storage
+  href:
+    type: string
+    format: uri
+    description: web uri of the storage instance
+  storageType:
+    type: string
+    enum:
+      - nextcloud
+    description: |-
+      The type of the storage resource. Currently only nextcloud storages are supported.
+  createdAt:
+    type: string
+    format: date-time
+    description: Time of creation
+  updatedAt:
+    type: string
+    format: date-time
+    description: Time of the most recent change to the storage
+  _links:
+    type: object
+    required:
+      - self
+    properties:
+      self:
+        allOf:
+          - $ref: './link.yml'
+          - description: |-
+              This storage resource
+
+              **Resource**: Storage
```

Comment:
`readOnly` properties are not declared.

Reply:
well, this model is currently used only in a `GET` endpoint, there is no usage in a `POST`, `PATCH` or `PUT`. Hence it is `readOnly` by default. The additional property should IMHO only be used, when a model is used for reading and writing, to differentiate the properties between both scopes.